packages/babel-helpers/src/helpers/iterableToArrayLimitLoose.ts

```diff
@@ -0,0 +1,23 @@
+/* @minVersion 7.0.0-beta.0 */
+
+export default function _iterableToArrayLimitLoose<T>(
+  arr: Array<T>,
+  i: number,
+) {
+  var iterator: Iterator<T> & Function =
+    arr &&
+    ((typeof Symbol !== "undefined" && arr[Symbol.iterator]) ||
+      (arr as any)["@@iterator"]);
+  if (iterator == null) return;
+
+  var _arr: T[] = [];
+  var step;
+  for (
+    iterator = iterator.call(arr);
+    arr.length < i && !(step = iterator.next()).done;
+
+  ) {
+    _arr.push(step.value);
+  }
```

Comment:
```suggestion
  iterator = iterator.call(arr);
  while (arr.length < i && !(step = iterator.next()).done) {
    _arr.push(step.value);
  }
```

we can also slightly improve readability here, to avoid the confusing empty for "update" segment.